tests/scripts/run_generators.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# ANSI colors
+BOLD_GREEN='\033[1;32m'
+BOLD_RED='\033[1;31m'
+ANSI_RESET='\033[0m'
+SUCCESS_PREFIX="${BOLD_GREEN}SUCCESS:${ANSI_RESET}"
+FAILURE_PREFIX="${BOLD_RED}FAILED:${ANSI_RESET}"
+
+TMP_DIR="tests/generators/tmp/"
+GOLDEN_DIR="tests/generators/golden/"
+
+FAILURE_COUNT=0
+check_result() {
+  local suffix=$1 # One of: js, py, dart, lua, php
+  local tmp_filename="${TMP_DIR}generated.$suffix"
+  if [ ! -f $tmp_filename ]; then
+    echo "File $tmp_filename not found!"
+    FAILURE_COUNT=$((FAILURE_COUNT+1))
```

Comment:
Should this return if the file doesn't exist?

Reply:
Changed to do the next steps if the file is found, and otherwise simply report the error.